azure-pipelines.yml

```
@@ -12,6 +12,12 @@ jobs:
         containerImage: norionomura/swiftlint:swift-4.2.0
       swift421:
         containerImage: norionomura/swiftlint:swift-4.2.1
+      swift422:
+        containerImage: norionomura/swiftlint:swift-4.2.2
+      swift423:
+        containerImage: norionomura/swiftlint:swift-4.2.3
+      swift50:
+        containerImage: norionomura/swift:5.0
```

Is there a reason to use this image over the official one?

```suggestion
        containerImage: swift:5.0
```